spec/models/collection_spec.rb

```diff
@@ -58,4 +58,52 @@
       expect(collection.valid?).to be(false)
     end
   end
+
+  context 'when validating druid' do
+    # let(:client) { instance_double(Dor::Services::Client, object: object_client) }
+    let(:object_client) { instance_double(Dor::Services::Client::Object) }
+
+    before do
+      allow(Settings).to receive(:validate_collection_druid).and_return(true)
+      # allow(Dor::Services::Client).to receive(:configure).and_return(client)
```

Comment:
Cruft on lines 63 and 68 or is that worth keeping around?

Reply:
Cruft. Removing.